examples/style/icon.html

```diff
@@ -72,7 +59,14 @@ <h4 class="as-subheader as-mb--12">
         carto.auth.setDefaultCredentials({ username: 'public' });
         const deckMap = carto.viz.createMap();
 
-        const style = carto.viz.style.icon('./boat.png');
+        /*
+          In this case this image is just a .png as a relative url (you can use of course an absolute url...)
+          and its size is specified in pixels
+        */
+        const style = carto.viz.style.icon('./boat.png', {
```

Comment:
I would use a different layer/icon combination. The world_ports layer has too many features within a huge extension/bounds and the icons are overlapped. The icon details are not appreciated at 24x24 size. Maybe we can use a layer with a smaller extension and a more simple/colorful icon with a greater size.

Reply:
Agree, I'm gonna merge this and prepare a smaller PR with it. Suggestions are welcome!